include/SFML/Window/Event.hpp

```diff
@@ -299,6 +299,17 @@ class SFML_WINDOW_API Event
     template <typename TEventSubtype>
     [[nodiscard]] const TEventSubtype* getIf() const;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Apply a visitor to the event
+    ///
+    /// \param visitor The visitor to apply
+    ///
+    /// \return The result of applying the visitor to the event
```

Comment:
This should be a bit more descriptive. What kind of results can the user expect?

Reply:
"The result of applying the visitor to the event" is the description of the return type, since the user gets to decide this there is no other way to describe it.

Reply:
One can still be more precise in specifying what is returning here, as in the result as specified by the visitors implementation.

Reply:
But... isn't that just saying the same thing just with different words? 😁 

Not to mention, maybe the implementation doesn't even specify the return type also just relying on it being automatically deduced.

Reply:
Yes, but right now it's very unclear how it happens, unless you go check under the hood. The result just seems to "spring into existence" and it's not clear that it's defined by the provided visitor, it might as well be defined by the event or derived from cosmic rays.

I think the difference in understanding comes from the point that "applying the visitor to the event" communicates to you a clear concept, while it might not be as clear to others.

Reply:
I am open to suggestions for how the whole line should be written verbatim so there is no ambiguity. 😁